src/Tools/GenerateGlobalAnalyzerConfigs/Program.cs

```diff
@@ -521,5 +554,12 @@ public Assembly LoadFromPath(string fullPath)
                 return Assembly.LoadFrom(fullPath);
             }
         }
+
+        private enum AnalysisMode
+        {
+            Default,
+            AllDisabled,
+            AllEnabled,
```

Comment:
@gewarren @mikadumont Any feedback on the customer facing values for `AnalysisMode` property? Other options that come to my mind are:

`Default, AllDisabledByDefault, AddEnabledByDefault`

OR

`Default, OptOut, OptIn`

OR

`Default, Conservative, Aggressive`

any other suggestions?